ring/ring.go

```diff
@@ -513,27 +515,26 @@ func (r *Ring) GetReplicationSetForOperation(op Operation) (ReplicationSet, erro
 }
 
 // countTokens returns the number of tokens and tokens within the range for each instance.
-// The ring read lock must be already taken when calling this function.
-func (r *Ring) countTokens() (map[string]uint32, map[string]uint32) {
+func countTokens(ringDesc *Desc, ringTokens []uint32, ringInstanceByToken map[uint32]instanceInfo) (map[string]uint32, map[string]uint32) {
```

Comment:
Shall we make this a method on `*Desc`, without extra arguments? Calling `desc.GetTokens()` and `ringDesc.getTokensInfo()` won't be as efficient, but I don't think we need to aim for efficiency when displaying web page.

Reply:
Agreed, done within 816b94b